contrib/win32/win32compat/win32_usertoken_utils.c

```diff
@@ -460,8 +460,11 @@ AddSidMappingToLsa(PUNICODE_STRING domain_name,
 			error("LsaManageSidNameMapping failed with ntstatus: %d \n", status);
 	}
 
-	if (p_output)
-		LsaFreeMemory(p_output);
+	if (p_output) {
+		status = LsaFreeMemory(p_output);
+		if (status != STATUS_SUCCESS)
+			debug3("LsaFreeMemory failed with ntstatus: %d \n", status);
```

Comment:
\n is not required.. If you have it then log file will show \n instead of treating it as new line.

Reply:
please fix in all the places in this file.

Reply:
fixed in all debug related strings